src/Symfony/Component/Messenger/Transport/AmqpExt/AmqpSender.php

```diff
@@ -41,7 +42,11 @@ public function send(Envelope $envelope): Envelope
     {
         $encodedMessage = $this->serializer->encode($envelope);
 
-        $this->connection->publish($encodedMessage['body'], $encodedMessage['headers']);
+        try {
+            $this->connection->publish($encodedMessage['body'], $encodedMessage['headers']);
+        } catch (\AMQPException $e) {
+            throw new TransportException('Current transport was not able to send given message, please try again');
```

Comment:
please add $e as $previous argument

Reply:
Sure! Which code should i give then? `0` as default?